config/config.go

```diff
@@ -41,7 +41,8 @@ type KafkaConfig struct {
 	SecClientKey                  string   `envconfig:"KAFKA_SEC_CLIENT_KEY"                  json:"-"`
 	SecClientCert                 string   `envconfig:"KAFKA_SEC_CLIENT_CERT"`
 	SecSkipVerify                 bool     `envconfig:"KAFKA_SEC_SKIP_VERIFY"`
-	CantabularMetadataExportTopic string   `envconfig:"CANTABULAR_METADATA_EXPORT_TOPIC"`
+	CantabularCSVCreatedTopic     string   `envconfig:"CANTABULAR_CSV_CREATED_TOPIC"`
```

Comment:
just a suggestion, maybe we should set the prefix `KAFKA_` for all the kafka related secrets? I would suggest changing this env var to `KAFKA_TOPIC_CANTABULAR_CSV_CREATED` (I followed this convention in `dp-cantabular-csv-exporter`)